app/Services/NetworkDataService.php

```diff
@@ -26,7 +27,12 @@ public function __construct(Client $client)
 
     public function updateNetworkData(): void
     {
-        $data = json_decode($this->client->get(self::NETWORK_DATA_URL)->getBody(), true);
+        try {
+            $data = json_decode($this->client->get(self::NETWORK_DATA_URL)->getBody(), true);
```

Comment:
Would recommend you consider the HTTP facade - the graceful 400+ error handling should work well in this context without having to try catch https://laravel.com/docs/7.x/http-client